menu_page.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="style.css"> 
+		<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+		<script src="http://code.jquery.com/jquery-2.1.3.min.js"></script>
+		<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+	</head>
+	<body>
+		<!-- Collin Kotz
+			.html
+			INFO2439
+			Thoendel
+			10/14/19
+        -->
+        
+		<div data-role="page" id="menu_page" data-title="The MCC Sage Bistro">
+		    <div data-role="header" data-postion="fixed">
+                <img src="MCC_Sage_Resources/Sage-Bistro-Logo-2017-transparenent-small.png" alt="logo" height="89" width="200">
+                <h1>Menu</h1>
+                
+                <div data-role="navbar">
+                    <ul>
+                        <li><h1><a href="home_page.html">Home</a></h1></li>
+                        <li><h1><a href="google_map_sage_bistro.html">Where To Find Us</a></h1></li>
+                        <li><h1><a href="survey_page.html">Survey</a></h1></li>
+                        <li><h1><a href="reservations_page.html">Reservations</a></h1></li>
+                        <li><h1><a href="menu_page">Menu</a></h1></li>
+                    </ul>
+                </div>
```

Comment:
This menu systems is nice and consistent! It does get pretty smooshed on a small screen.